### PR TITLE
pythonPackages.sslib: init at 0.2.0

### PR DESCRIPTION
**Pull request by jqueiroz**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by FRidh** [comment]
@GrahamcOfBorg build pythonPackages.sslib python3Packages.sslib

File: pkgs/development/python-modules/sslib/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "sslib";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
```
Comment (dotlambda) [inline pkgs/development/python-modules/sslib/default.nix]: Why not `fetchPypi`?

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -580,6 +580,8 @@ in {
 
   spglib = callPackage ../development/python-modules/spglib { };
 
+  sslib = disabledIf (!isPy3k) (callPackage ../development/python-modules/sslib { });
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: disabledIf is intended for non-`buildPythonPackage` expressions. Pass `disabled = !isPy3k;` instead.

File: pkgs/development/python-modules/sslib/default.nix (hunk 1 above)
Comment (jqueiroz) [reply pkgs/development/python-modules/sslib/default.nix]: Changed, thanks.

File: pkgs/top-level/python-packages.nix (hunk 2 above)
Comment (jqueiroz) [reply pkgs/top-level/python-packages.nix]: Fixed, thanks.

File: pkgs/development/python-modules/sslib/default.nix [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchPypi, buildPythonPackage, isPy3k }:
+
+buildPythonPackage rec {
+  pname = "sslib";
+  version = "0.2.0";
+  format = "wheel";
```
Comment (dotlambda) [inline pkgs/development/python-modules/sslib/default.nix]: I saw only now that PyPI only has wheels. We prefer building from source. Please ask upstream to publish a source tarball or go back to using `fetchFromGitHub`. Sorry!

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/sslib/default.nix (hunk 3 above)
Comment (jqueiroz) [reply pkgs/development/python-modules/sslib/default.nix]: Thanks, I added the missing source archive to PyPI and updated the derivation accordingly.

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python3Packages.sslib

**Comment by dotlambda** [comment]
> Ran 0 tests in 0.000s

Please provide an appropriate `checkPhase` or - if there are no tests - set `doCheck = false` and add a comment with the reason.

**Comment by jqueiroz** [comment]
> > Ran 0 tests in 0.000s
> 
> Please provide an appropriate `checkPhase` or - if there are no tests - set `doCheck = false` and add a comment with the reason.

Fixed, thanks.

**Pull request merged**